daemon/image_delete.go

```diff
@@ -124,9 +116,8 @@ func (daemon *Daemon) DeleteImage(eng *engine.Engine, name string, list *[]types
 				Deleted: img.ID,
 			})
 			daemon.EventsService.Log("delete", img.ID, "")
-			eng.Job("log", "delete", img.ID, "").Run()
```

Comment:
why this is here if no handler (nor catchall handler) is registered for "log" job?